.local/bin/browser_bookmarks.sh

```diff
@@ -83,7 +86,7 @@ open_bookmark() {
     URL=$(grep "^$SELECTION=" "$URLQUERY_FILE" | cut -d= -f2-)
     [ -z "$URL" ] && notify-send "Bookmark not found." && exit 1
     case "$URL" in
-    *"search"*|*"wiki"*)
+    *"search"*|*"wiki"*|*"packages"*)
```

Comment:
The comment at the very top could be appended at this line, like so:
`*"search"*|*"wiki"*|*"packages"*) #For additional search function, add keywords next to this`

This makes it more clear, and also removes the 2 comments at the top of the script (because it's hard for a random user to understand which lines you refer)

Reply:
Done.